### DISCUSSION TITLE
Cntrl-v on Windows 11 pastes instead of splits?

### DISCUSSION THREAD

**Discussion by jackmott** [helix-editor/helix#8882] [Q&A]
I was working through learning Helix and saw that Cntrl-v can be used to do a vertical split, but when I do that it seems to paste instead, as if the windows pasting functionality is taking precedence over Helix.  Is this a known issue? Is there a workaround?

**Comment by the-mikedavis** [comment]
Your terminal emulator is most likely taking over the C-v input to send pasted text. This may be something you can configure in the terminal emulator.
